acr.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
The script is very specific for Tasks environment and Linux only. I suggest we implement a go module/package to handle the auth flow which is compatible with docker . Similar to https://github.com/deislabs/oras/tree/master/pkg/auth/docker. It will also allow us to support windows in the future.

For the command interface, `acr` can continue to support username/password. If they're not specified, look for the auth info from .docker/config file.

Loop @sajayantony  @ehotinger  @shizhMSFT for comments.

Reply:
@northtyphoon we can implement it the same as Docker CLI. Will make a separate item for this.